.github/workflows/pull_request.yml

```diff
@@ -57,13 +57,10 @@ jobs:
       with:
         image: "openfoodfacts-server/backend:dev"
     - name: Fetch origin/main
-      # we need origin/main to have comparison linting work
       run: |
         git remote set-branches --add origin main
         git fetch --no-tags --prune --progress --no-recurse-submodules --depth=5 origin main
     - name: Restore taxonomies dates
```

Comment:
```suggestion
    - name: Restore taxonomies dates
      # here we first restore dates from git for taxonomies to avoid build them all
      # see https://stackoverflow.com/a/60984318/2886726
```